docs/store/datastore.md

```diff
@@ -175,58 +250,64 @@ profile = DatastoreProfileKafkaTarget(name="test_profile",bootstrap_servers="loc
+- `sasl_pass` &mdash; A string representing the password for SASL authentication, correlating with the `sasl_user`. It's tagged as private for security considerations.
+- `kwargs_public` &mdash; This is a dictionary (`Dict`) that holds a collection of key-value pairs used to represent settings or configurations deemed public. These pairs are subsequently passed as parameters to the underlying `kafka.KafkaProducer()` constructor. It defaults to `None`.
+- `kwargs_private` &mdash; This dictionary (`Dict`) is used to store key-value pairs, typically representing configurations that are of a private or sensitive nature. These pairs are subsequently passed as parameters to the underlying `kafka.KafkaProducer()` constructor. It defaults to `None`.
 
 ### Redis data store profile
 
-`profile = DatastoreProfileRedis(name="test_profile", endpoint_url="redis://11.22.33.44:6379", username="user", password="password")`
-`RedisNoSqlTarget(path="ds://test_profile/a/b")
-
+```
+profile = DatastoreProfileRedis(name="test_profile", endpoint_url="redis://11.22.33.44:6379", username="user", password="password")
+RedisNoSqlTarget(path="ds://test_profile/a/b")
+```
 
 ### S3 data store profile
 
-`profile = DatastoreProfileS3(name="test_profile")`
-`ParquetTarget(path="ds://test_profile/aws_bucket/path/to/parquet.pq")`
+
+```
+profile = DatastoreProfileS3(name="test_profile")
+ParquetTarget(path="ds://test_profile/aws_bucket/path/to/parquet.pq")
+```
 
 `DatastoreProfileS3` init parameters:
-- *name*: name of the profile
-- *endpoint_url* (optional): A string representing the endpoint URL of the S3 service. It's typically required for non-AWS S3-compatible services. If not provided, the default is `None`.
-- *force_non_anonymous* (optional): A string that determines whether to force non-anonymous access to the S3 bucket. The default value is `None`, meaning the behavior is not explicitly set.
-- *profile_name* (optional): A string representing the name of the profile. This might be used to refer to specific named configurations for connecting to S3. The default value is `None`.
-- *assume_role_arn* (optional): A string representing the Amazon Resource Name (ARN) of the role to assume when interacting with the S3 service. This can be useful for granting temporary permissions. By default, it is set to `None`.
-- *access_key* (optional): A string representing the access key used for authentication to the S3 service. It's one of the credentials parts when you're not using anonymous access or IAM roles. For privacy reasons, it's tagged as a private attribute, and its default value is `None`.
-- *secret_key* (optional): A string representing the secret key, which pairs with the access key, used for authentication to the S3 service. It's the second part of the credentials when not using anonymous access or IAM roles. It's also tagged as private for privacy and security reasons. The default value is `None`.
+- `name` &mdash; Name of the profile
```

Comment:
Can you provide the equal env parameter per variable? @alxtkr77

Reply:
env["AWS_ACCESS_KEY_ID"] = access_key
            env["AWS_SECRET_ACCESS_KEY"] = secret_key
            env["S3_ENDPOINT_URL"] = endpoint_url
            env["S3_NON_ANONYMOUS"] = force_non_anonymous
            env["AWS_PROFILE"] = profile_name
            env["MLRUN_AWS_ROLE_ARN"] = assume_role_arn

Reply:
added